PCbuild/python.props

```diff
@@ -180,6 +189,7 @@
 
     <!-- The version and platform tag to include in .pyd filenames -->
     <PydTag Condition="$(ArchName) == 'win32'">.cp$(MajorVersionNumber)$(MinorVersionNumber)-win32</PydTag>
+    <PydTag Condition="$(ArchName) == 'arm32'">.cp$(MajorVersionNumber)$(MinorVersionNumber)-win_arm</PydTag>
```

Comment:
Looking ahead to the future, do you think "ARM == ARM32" is going to hold for the long term? Or should we explicitly put `win_arm32` now? I'd prefer to be more explicit, unless there's a _lot_ of precedent for leaving the bitness out here.

Reply:
And I mean from a consumer point of view. Clearly Microsoft has decided that `$(Platform)=='ARM'` already, but that doesn't directly follow that we have to do the same.

Reply:
I'm glad you brought this up.  There was some win_arm code already in cpython, but it looks like it was there to support Windows CE.  That might be another reason to use win_arm32 here, to completely disambiguate it from both arm64 and arm on Windows CE.

Reply:
specifically the arm32 implementation currently relies on the following snippet in pc/pyconfig.h
It looks like support for Windows CE has been removed but there's a few comments still lingering.  If Windows CE is no longer supported then the PYD_PLATFORM_TAG will also need to be edited to match.  There's a doc snippet in Doc/whatsnew/3.5.rst that says the platform tag for Windows on ARM is win_arm as well.

#elif defined(_M_ARM)
#define COMPILER _Py_PASTE_VERSION("32 bit (ARM)")
#define PYD_PLATFORM_TAG "win_arm"
#else

Reply:
@pfmoore @zware Any thoughts on this? I'm +0 on using `win_arm32` rather than `win_arm`, but is there any chance people are relying on this branch in the code? I doubt it, so I think just changing it here is going to be fine, but I'd like another opinion.